ex1-tema2.js

```diff
@@ -7,4 +7,22 @@ function even_numbers (n){
 }
 even_numbers(10);
 
+//Tipăriți toate numerele impare  de la 0 la 10
```

Comment:
Acest fisier ar trebui sa nu fi in acest branch:
https://github.com/Emilia-Dumitru/Tema2-Js/network

Din graphul de mai sus se vede ca branchurile noi nu au fost create din main, ci din tema2-ex1. De aceea apare aici